src/Compute/Compute/Common/ValidateBase64EncodedString.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Commands.Compute.Common;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.Internal.Resources;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+
+namespace Microsoft.Azure.Commands.Compute.Common
+{
+    public static class ValidateBase64EncodedString
```

Comment:
This class looks very suitable for unit testing

Reply:
Added a test for the functionality in this class.